ContentPatcher/Framework/Api/ApiManagedValueForSingleScreen.cs

```diff
@@ -0,0 +1,100 @@
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using ContentPatcher.Framework.Commands;
+using ContentPatcher.Framework.Conditions;
+using ContentPatcher.Framework.Tokens;
+
+namespace ContentPatcher.Framework.Api
+{
+    /// <summary>A parsed string linked to the Content Patcher context for an API consumer, which assume they're always run on the same screen.</summary>
+    internal class ApiManagedValueForSingleScreen : IManagedValue
+    {
+        /*********
+        ** Fields
+        *********/
+        private readonly IManagedTokenString TokenString;
+
+        /// <summary>The context with which to update conditions.</summary>
+        private readonly IContext Context;
+
+        /// <summary>A contextual manager for the underlying conditions.</summary>
+        private readonly AggregateContextual Contextuals;
```

Comment:
`AggregateContextual` collects multiple `IContextual` values into a single group which can be updated and checked together. But in this case we only have one value, so I suggest just using the relevant methods on `this.TokenString` directly instead.